composeApp/src/commonMain/kotlin/sp/bvantur/tasky/event/data/EventRepositoryImpl.kt

```diff
@@ -46,5 +51,10 @@ class EventRepositoryImpl(
     }
 
     override suspend fun createEvent(event: Event): TaskyEmptyResult<CommunicationError> =
-        remoteDataSource.createEvent(event.asCreateEventRequest()) // TODO store data to database
+        remoteDataSource.createEvent(event.asCreateEventRequest()).onSuccess { data ->
+            localDataSource.saveEvent(data.asEventEntity())
+            data.attendees.forEach { attendee ->
+                localDataSource.saveAttendee(attendee.asAttendeeEntity())
+            }
+        }.asEmptyDataResult()
 }
```

Comment:
I was thinking of doing offline/syncing stuff later when we go over this logic of how to properly read data from data sources first.

Reply:
OK, I advise you to leave a comment for yourself here to add this functionality later. It will make it much easier on you when you need to recall what functionality you still need to implement.